pkg/plugins/policies/meshtls/plugin/v1alpha1/plugin.go

```diff
@@ -270,14 +266,7 @@ func configureParams(conf *api.Conf, cluster *envoy_cluster.Cluster) error {
 	return nil
 }
 
-func configure(
-	proxy *core_xds.Proxy,
-	listener *envoy_listener.Listener,
-	iface mesh_proto.InboundInterface,
-	inbound *mesh_proto.Dataplane_Networking_Inbound,
-	conf *api.Conf,
-	xdsCtx xds_context.Context,
-) (envoy_common.NamedResource, error) {
+func configure(proxy *core_xds.Proxy, listener *envoy_listener.Listener, iface mesh_proto.InboundInterface, inbound *mesh_proto.Dataplane_Networking_Inbound, conf api.Conf, xdsCtx xds_context.Context) (envoy_common.NamedResource, error) {
```

Comment:
Each arg on a separate line is easier to read no?

Reply:
yes, goland reformatted it, and I didn't catch that, updated